sdk/tables/data-tables/swagger/README.md

```diff
@@ -62,3 +60,30 @@ directive:
     transform: >
       $["x-ms-client-name"] = "etag";
 ```
+
+### Make Access Policy not required
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.SignedIdentifier
+  transform: >
+    $.required = ["Id"]
+```
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.AccessPolicy
+  transform: >
+    delete $.required
+```
```

Comment:
These two transforms is to fix a problem with the swagger in which it is misrepresenting what the service expects. It actually can take empty AccessPolicy and also a SignedIdentity with only an ID and no AccessPolicies